### ISSUE TITLE
Trouble binding HTTP Prefixes using the ENV variable

### ISSUE THREAD

**Issue by litch** [EventStore/EventStore#436]
If I start my cluster like this:

```
root@ip-10-0-0-146:/usr/lib/event-store# ./clusternode --http-prefixes=http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-146.ec2.internal:2113/
```

then everything works. :sparkles:

However, if I try to set that via an ENV variable

```
root@ip-10-0-0-146:/usr/lib/event-store# export EVENTSTORE_HTTP_PREFIXES=http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-146.ec2.internal:2113/
root@ip-10-0-0-146:/usr/lib/event-store# ./clusternode
```

Then I get the following error

```
Error while parsing options:
The value http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-146.ec2.internal:2113/ could not be converted to String[]

```

**Comment by litch** [comment]
FWIW, I have also tried enclosing the ENV variable data in quotes.

**Comment by pgermishuys** [comment]
Hi @litch,
Will get back to you as soon I as figure out what gremlin has crept into the osx/linux side of things. On windows, this is working as expected.

**Comment by pgermishuys** [comment]
@litch, here is the output from Windows and OSX. About to check on Ubuntu.

Windows:
```
[02228,01,20:29:55.598]
ES VERSION:               0.0.0.0 (Unknown/Unknown, Unknown)
OS:                       Windows (Microsoft Windows NT 6.2.9200.0)
RUNTIME:                  .NET 4.0.30319.34014 (64-bit)
GC:                       3 GENERATIONS
THREADPOOL:               Worker Threads (Min: 2, Max: 32767) - IOCP Threads: (Min: 2, Max: 1000)
LOGS:                     C:\Users\vagrant\AppData\Local\Temp\EventStore\2015-03-27_20.29.55.160983-Node2113-logs

MODIFIED OPTIONS:

        WHAT IF:                  True (Command Line)
        HTTP PREFIXES:            http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-146.ec2.internal:2113/ (Environment Variable)

DEFAULT OPTIONS:
```
OSX:
```
32948,01,20:31:52.937]
ES VERSION:               0.0.0.0 (dev/d61321ab2fdfcd71abf7a9f4585178ee45fac342, Mon, 9 Mar 2015 16:35:27 +0200)
OS:                       MacOS (Unix 14.1.0.0)
RUNTIME:                  3.99.0 (master/cfc0d6f Thu Feb 12 04:44:30 SAST 2015) (64-bit)
GC:                       2 GENERATIONS (Desktop Mode - Interactive)
THREADPOOL:               Worker Threads (Min: 10, Max: 800) - IOCP Threads: (Min: 16, Max: 32)
LOGS:                     /var/folders/w0/0x3y5m595bb4f1gs_h9jd2pr0000gn/T/EventStore/2015-03-27_20.31.52.493141-Node2113-logs

MODIFIED OPTIONS:

	WHAT IF:                  True (Command Line)
	HTTP PREFIXES:            http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-146.ec2.internal:2113/ (Environment Variable)

DEFAULT OPTIONS:
```
I did pick up an issue though which will get fixed asap.
The code responsible for getting the environment variables isn't trying to get the value for EVENTSTORE_HTTP_PREFIXES but rather EVENTSTORE_Http_Prefixes

**Comment by litch** [comment]
Very strange.  What are you using to invoke this on the mac?  I can't get it to run on the mac with ENV variables either:

```
☹ ~/Downloads/EventStore-OSS-Mac-v3.0.3% EVENTSTORE_HTTP_PREFIXES=http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-97.ec2.internal:2113/ ./clusternode 
Error while parsing options:
The value http://ec2-xx-xx-xx-xx.compute-1.amazonaws.com:2113/,http://ip-10-0-0-97.ec2.internal:2113/ could not be converted to String[]

Options:
Usage: EventStore.ClusterNode options
...
```

**Comment by litch** [comment]
Also, for the record, I am now seeing this with `EVENTSTORE_GOSSIP_SEED` as well: 

```
cat /etc/sv/event-store/env/EVENTSTORE_GOSSIP_SEED 
ip-10-0-0-38.ec2.internal,ip-10-0-0-112.ec2.internal
```
```
Error while parsing options:
The value ip-10-0-0-38.ec2.internal,ip-10-0-0-112.ec2.internal could not be converted to IPEndPoint[]
```

**Comment by litch** [comment]
Oops, nevermind that - I had misformatted things.  
```
10.0.0.38:2113,10.0.0.112:2113
```
Worked just fine

**Comment by pgermishuys** [comment]
I do apologise for the delay in getting back to you on this particular issue.
In v3.0.0 up until v3.0.3 we are still using the slightly buggy version of parsing options. 

We have fixed numerous issues since then and this was when we moved the options parsing into it's own library called [Rags](https://github.com/EventStore/EventStore/tree/dev/etc/EventStore.Rags).

Unfortunately I don't have a work around for you now, but to wait until the version that uses Rags makes it into a release.
